totp/cli.py

```diff
@@ -116,5 +116,21 @@ def add_uri(path, uri):
 def _cmd_show(args):
     totp.generate_token(args.identifier, seconds=args.offset_seconds)
 
+@subcommand(
```

Comment:
I think it makes a lot more sense for this whole feature to just be implemented as an argument to the existing show command. Eg: have a `--nocopy` flag for it.

Otherwise this is just too much duplicate code that just does the same.

Reply:
Fair enough.
I added it as an argument instead.